web/src/stream_popover.js

```diff
@@ -604,6 +610,22 @@ export function register_stream_handlers() {
 }
 
 export function register_topic_handlers() {
+    // Toggle topic visibility_policy (currently in dev environment)
+    $("body").on("click", ".sidebar-popover-toggle-topic-visibility", (e) => {
+        const stream_id = topic_popover_stream_id(e);
+        if (!stream_id) {
+            return;
+        }
+
+        const topic = $(e.currentTarget).attr("data-topic-name");
+        const new_visibility_policy =
+            user_topics.get_topic_visibility_policy(stream_id, topic) === 2 ? 0 : 2;
```

Comment:
We shouldn't ever use raw numbers that translate to logical states like the 0/2 in code like this -- it's impossible to read. See `web/src/settings_config.ts` for some examples of a good pattern; I see we actually already define the right structure in `user_topics.js`:

```
export const visibility_policy = {                                                                      
    INHERIT: 0,                                                                                         
    MUTED: 1,                                                                                           
    UNMUTED: 2,                                                                                         
    FOLLOWED: 3,                                                                                        
};                                                                                                      
```